shell/browser/atom_browser_client.cc

```diff
@@ -521,48 +521,53 @@ void AtomBrowserClient::AppendExtraCommandLineSwitches(
 
   std::string process_type =
       command_line->GetSwitchValueASCII(::switches::kProcessType);
-  if (process_type != ::switches::kRendererProcess)
-    return;
 
-  // Copy following switches to child process.
-  static const char* const kCommonSwitchNames[] = {
-      switches::kStandardSchemes,      switches::kEnableSandbox,
-      switches::kSecureSchemes,        switches::kBypassCSPSchemes,
-      switches::kCORSSchemes,          switches::kFetchSchemes,
-      switches::kServiceWorkerSchemes, switches::kEnableApiFilteringLogging};
-  command_line->CopySwitchesFrom(*base::CommandLine::ForCurrentProcess(),
-                                 kCommonSwitchNames,
-                                 base::size(kCommonSwitchNames));
+  if (process_type == ::switches::kUtilityProcess ||
+      process_type == ::switches::kRendererProcess) {
+    // Copy following switches to child process.
+    static const char* const kCommonSwitchNames[] = {
+        switches::kStandardSchemes,      switches::kEnableSandbox,
+        switches::kSecureSchemes,        switches::kBypassCSPSchemes,
+        switches::kCORSSchemes,          switches::kFetchSchemes,
+        switches::kServiceWorkerSchemes, switches::kEnableApiFilteringLogging};
```

Comment:
`kEnableApiFilteringLogging` only needs to go to the renderer, not the utility process FWIW